tools/topology/topology2/include/components/aria.conf

```diff
@@ -43,14 +43,6 @@ Class.Widget."aria" {
 			put 0
 		}
 		max 4096
-		Object.Base.data. 1 {
-			# attenuation for the aria module
-			bytes "0x53, 0x4f, 0x46, 0x34, 0x01, 0x00, 0x00, 0x00,
-			0x4, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x08,
-			0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-			0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-			0x01, 0x00, 0x00, 0x00"
-		}
```

Comment:
When looking at the quite similar hex values of the blob it might be that this would have worked if I had first done the size_t vs. uint32_t fix to code that failed on 64 bit build. I think I should rewrite the commit message to say it's a blob ABI update and blobs added for all allowed configurations of Aria.

Reply:
Done